.taskcluster.yml

```diff
@@ -0,0 +1,52 @@
+version: 0
+
+metadata:
+  name: Normandy
+  description: Normandy CI Tasks
+  owner: "{{ event.head.user.email }}"
+  source: "{{ event.head.repo.url }}"
+
+tasks:
+  - metadata:
+      name: normandy:decision
+      description: Normandy decision task
+      owner: "{{ event.head.user.email }}"
+      source: "{{ event.head.repo.url }}"
+    provisionerId: "{{ taskcluster.docker.provisionerId }}"
+    workerType: "{{ taskcluster.docker.workerType }}"
+    scopes:
+      - queue:create-task:aws-provisioner-v1/github-worker
+    payload:
+      env:
```

Comment:
Everything looks good. I was just wondering if it would make sense to set this explicitly to true or false, just for clarity.